src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/formats/html/resources/standard.properties

```diff
@@ -121,6 +121,8 @@ doclet.New_Elements=New {0}
 doclet.New_Elements_Release_Column_Header=Added in
 doclet.New_Label=New
 doclet.Preview_API=Preview API
+doclet.Preview_API_Checkbox_Label=Show preview API for:
```

Comment:
Should "Preview API" be capitalized as a Proper Noun?

Reply:
You surely know better about English capitalization rules than me, but I notice that "preview API" is commonly written in lower case, for instance here: https://openjdk.org/jeps/12. I'll leave it like this for now, feel free to raise the issue if you think it should be changed.